src/main/scala/com/softwaremill/realworld/common/CustomHttpAuthorization.scala

```diff
@@ -0,0 +1,51 @@
+package com.softwaremill.realworld.common
+
+import sttp.model.HeaderNames
+import sttp.model.headers.WWWAuthenticateChallenge
+import sttp.tapir.{Codec, CodecFormat, DecodeResult, EndpointInput, Mapping, Schema, header}
+
+// This authorization allows the use of both the "Bearer" header and the "Token" parameter at the beginning of the token.
+object CustomHttpAuthorization {
+
+  type StringListCodec[T] = Codec[List[String], T, CodecFormat.TextPlain]
+
+  def authorizeBearerAndTokenHeaders[T: StringListCodec]: EndpointInput.Auth[T, EndpointInput.AuthType.Http] = {
```

Comment:
**suggestion:** Since it's authentication, not authorization, this `authorizeBearerAndTokenHeaders` name is also inaccurate. I'd change it to just `authBearerAndTokenHeaders`.